### PR TITLE
EDSC-1597: Add start and end labels to granule temporal ranges

### PR DESCRIPTION
**Pull request by jennywagg**

### CODE REVIEW

File: app/assets/stylesheets/modules/panel_list.css.scss [hunk 1]
```diff
@@ -85,19 +85,49 @@ $panel-list-thumbnail-container-size: $panel-list-thumbnail-size + 4px;
...
+      display: inline-block;
+      font-size: .9em;
+    }
+
+    .temporal-start h5 {
+      color: rgba(39, 174, 96,1);
+      width: 50px;
+    }
+
+    .temporal-start p{
```
Comment (mreese84) [inline app/assets/stylesheets/modules/panel_list.css.scss]: Small nitpick. This needs a space after the `p`

File: app/assets/stylesheets/modules/panel_list.css.scss [hunk 2]
```diff
@@ -85,19 +85,49 @@ $panel-list-thumbnail-container-size: $panel-list-thumbnail-size + 4px;
...
+      display: inline-block;
+      font-size: .9em;
+    }
+
+    .temporal-end h5 {
+      color: rgba(192, 57, 43,1);
+      width: 50px;
+    }
+
+    .temporal-end p{
```
Comment (mreese84) [inline app/assets/stylesheets/modules/panel_list.css.scss]: Another small nitpick. This needs a space after the `p`

File: app/assets/stylesheets/modules/panel_list.css.scss [hunk 3]
```diff
@@ -85,19 +85,49 @@ $panel-list-thumbnail-container-size: $panel-list-thumbnail-size + 4px;
...
-
-    p {
-      margin: 0;
-      padding: 3px 10px;
-    }
-
     .temporal-start {
       background: rgba(39, 174, 96,0.1);
+      text-align: justify;
+      overflow: hidden
```
Comment (mreese84) [inline app/assets/stylesheets/modules/panel_list.css.scss]: This needs a semicolon after "hidden".

File: app/assets/stylesheets/modules/panel_list.css.scss [hunk 4]
```diff
@@ -85,19 +85,49 @@ $panel-list-thumbnail-container-size: $panel-list-thumbnail-size + 4px;
...
     }
 
     .temporal-end {
       background: rgba(192, 57, 43,0.1);
       border-bottom: 1px solid rgba(192, 57, 43,0.2);
+      overflow: hidden
```
Comment (mreese84) [inline app/assets/stylesheets/modules/panel_list.css.scss]: This needs a semicolon after "hidden" too.

File: app/assets/javascripts/models/data/granule.coffee [hunk 5]
```diff
@@ -37,7 +37,8 @@
       return time_start unless time_end?
       return time_end unless time_start?
 
-      "<p class=\"temporal-start\">#{time_start}</p><p class=\"temporal-end\">#{time_end}</p>"
+#      "<p class=\"temporal-start\"><h5 class=\"temporal-start-label\">START</h5>#{time_start}</p><p class=\"temporal-end\"><h5 class=\"temporal-end-label\">END</h5>#{time_end}</p>"
+      "<div class=\"temporal-start\"> <h5>START</h5><p>#{time_start}</p></div><div class=\"temporal-end\"> <h5>END</h5><p>#{time_end}</p></div>"
```
Comment (mreese84) [inline app/assets/javascripts/models/data/granule.coffee]: @jsiarto, how do we feel about using a heading element for this? I consider this a label, not a heading from a semantic standpoint. Any thoughts on your end?

File: app/assets/javascripts/models/data/granule.coffee [hunk 6]
```diff
@@ -37,7 +37,8 @@
       return time_start unless time_end?
       return time_end unless time_start?
 
-      "<p class=\"temporal-start\">#{time_start}</p><p class=\"temporal-end\">#{time_end}</p>"
+#      "<p class=\"temporal-start\"><h5 class=\"temporal-start-label\">START</h5>#{time_start}</p><p class=\"temporal-end\"><h5 class=\"temporal-end-label\">END</h5>#{time_end}</p>"
```
Comment (mreese84) [inline app/assets/javascripts/models/data/granule.coffee]: You probably just commented this out while you were experimenting, just make sure to delete the line when you're done.

**Comment by mreese84** [comment]
@jennywagg, some small comments to address from a code style standpoint, but other than that it looks great. Just clean up the code a tad and this will be wrapped up.

**Comment by jennywagg** [comment]
@mreese84 I shouldn't have been in a hurry to push, thanks for pointing those things out! Fixed them.

**Pull request merged**